spec/models/test_track/session_spec.rb

```diff
@@ -118,6 +118,18 @@
     end
 
     context "cookies" do
+      it "uses the default cookie name if not configured" do
```

Comment:
Should we have a test where the cookie name is configured? Or is that testing the framework?